src/types/profile-derived.js

```diff
@@ -158,3 +158,27 @@ export type LocalTrack =
 
 export type Track = GlobalTrack | LocalTrack;
 export type TrackIndex = number;
+
+/**
+ * Type that holds the values of personally identifiable information that user
+ * wants to remove.
+ */
+export type PII = {
```

Comment:
I'm finding the implementation logic a bit hard to follow, I think because some of the terms in this type are a bit overloaded and unclear. Also, this isn't necessarily all PII, it's also just information that is hidden and probably not useful.

What do you think about:

```
export type RemoveProfileInformation = {
  threads: ThreadIndex[],
  threadsWithScreenshots: ThreadIndex[],
  timeRange: StartEndRange | null,
  removeNetworkUrls: boolean,
  removeAllUrls: boolean,
  removeExtensions: boolean,
};
```

All of the boolean values would start with `removeXXX`, that way it's clear it's a boolean and not a reference on what to remove.

Reply:
I'd even propose `shouldRemoveXXX`